conveyal/analysis-ui
# Delete regional analyses from the Java API

A previous refactor to Regional Analyses changed it to use the Next.js API for deletion from MongoDB. This caused analyses to keep running because the job was not also stopped in the Broker. This PR changes it back to delete from the Java API.

## How to test

1. Create a regional analysis. While it is running, delete it. 
2. Ensure that the job is stopped from the Admin panel and that there are no errors.
